persist-cli/src/main/java/io/ballerina/persist/cmd/Migrate.java

```diff
@@ -0,0 +1,822 @@
+                    }
+                }
+                break;
+
+            case ADD_READONLY:
+                for (String entity : map.keySet()) {
+                    if (!addedEntities.contains(entity)) {
+                        for (FieldMetadata field : map.get(entity)) {
+                            String primaryKey = field.getName();
+                            String addReadOnlyTemplate = "ALTER TABLE %s ADD PRIMARY KEY (%s);";
+
+                            queries.add(String.format(addReadOnlyTemplate, entity, primaryKey));
+                        }
+                    }
+                }
+                break;
+
+            default:
+                break;
+        }
+    }
+
+    // Convert map of ForeignKey lists to a MySQL query
+    public static void convertFKMapToQuery(queryTypes type, Map<String, List<ForeignKey>> map, List<String> queries) {
+        if (Objects.requireNonNull(type) == queryTypes.ADD_FOREIGN_KEY) {
+            for (String entity : map.keySet()) {
+                for (ForeignKey foreignKey : map.get(entity)) {
+
+                    String foreign_key_name = foreignKey.getName();
+                    String child_column_name = foreignKey.getColumnName();
+                    String reference_table_name = foreignKey.getReferenceTable();
+                    String reference_column_name = foreignKey.getReferenceColumn();
+                    String addForeignKeyTemplate = "ALTER TABLE %s ADD CONSTRAINT %s FOREIGN KEY (%s) REFERENCES %s(%s);";
+
+                    queries.add(String.format(addForeignKeyTemplate, entity, foreign_key_name, child_column_name,
+                            reference_table_name, reference_column_name));
+                }
+            }
+        }
+    }
+
+    // Convert Java data type to MySQL data type
+    public static String getDataType(String dataType) {
+        String resultType;
+
+        switch (dataType) {
+            case "string":
+                resultType = "VARCHAR(191)";
+                break;
+
+            case "int":
+                resultType = "INT";
+                break;
+
+            case "long":
+                resultType = "BIGINT";
+                break;
+
+            case "float":
+                resultType = "FLOAT";
+                break;
+
+            case "double":
+                resultType = "DOUBLE";
+                break;
+
+            case "boolean":
+                resultType = "TINYINT(1)";
+                break;
+
+            case "Date":
+            case "LocalDate":
+                resultType = "DATE";
+                break;
+
+            case "LocalTime":
+                resultType = "TIME";
+                break;
+
+            case "LocalDateTime":
+            case "time:Civil":
+                resultType = "DATETIME";
+                break;
+
+            case "Blob":
+            case "byte[]":
+                resultType = "BLOB";
+                break;
+
+            case "Clob":
```

Comment:
I think that this type is not supported yet. You can check this function. https://github.com/ballerina-platform/persist-tools/blob/main/persist-cli/src/main/java/io/ballerina/persist/nodegenerator/syntax/utils/SqlScriptUtils.java#L201

Reply:
Shall I remove any data type that is not available in that function then? Or just BLOB and CLOB?

Reply:
Yes, we can remove those. 
@daneshk Shall we remove those?